src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3438,6 +3438,9 @@
   <data name="CopyFailureFileToLarge" xml:space="preserve">
     <value>The following items are too large to be copied to this drive</value>
   </data>
+  <data name="FormatYourDrive" xml:space="preserve">
+    <value>Format your drive</value>
```

Comment:
```suggestion
    <value>Format drive</value>
```
What do you think?

Reply:
I remember someone settled a debate about whether Files should be more friendly or neutral, but I forget how it ended. If we want the latter, we should remove `your`

Reply:
I think it's best to decide each case separately. In this scenario, it can be someone else's drive so "your" might be out of place.